Popup feature

tobielf opened the pull request:
When click on the cell will display a popup menu to display the value
inside the cell(both vertices and edges).

tobielf committed:
Add pop up feature.

When click on the cell will display a popup menu to display the value
inside the cell(both vertices and edges).

tobielf committed:
Merge branch 'PLPVisualizer' into popup-feature

hariiyerasu1729 commented:
When the window is resized to max size, the graph needs to be scrolled. In that case, the Popup menu appears on the original position of the vertex before scrolling. As in, the Popupmenu is based on the window position, not the Graph.

tobielf commented:
I will convert it to component position.

tobielf committed:
Convert the position from Windows Position to Component Position

The popup menu was based on the window position, which will lead to a
drifting under full screen model. Changed it to Component Position to
fixed the issue.
